source/sync.txt

```diff
@@ -8,21 +12,129 @@
+
+.. image:: /images/plp/Spot_AzureBlue_Sync_Tech_RealmSync.png
+   :alt: Atlas Device Sync Illustration
+
+.. kicker:: What It Is
+
+Key Features
+------------
+
+.. card-group::
+   :columns: 3
+   :style: compact
+
+   .. card::
+      :headline: Realm in the Front, MongoDB in the Back
+      :icon: /images/icons/General_FEATURES_IntelligentEdge3x.png
+      :icon-alt: An icon depicting a mobile phone connecting with the cloud
+
+      {+sync+} is a bridge between client apps using the :realm:`Realm SDKs </>`
+      and a MongoDB instance running in Atlas. Realm is a lightweight database
+      optimized for mobile development.
+      
+   .. card::
+      :headline: Offline First
+      :icon: /images/icons/Technical_REALM_Offline3x.png
+      :icon-alt: An icon depicting network loss
+
+      Realm Database and {+sync-short+} seamlessly handle intermittent
+      connectivity so users can continue using your app regardless of their
+      current network status.
+
+   .. card::
+      :headline: Conflict Resolution
+      :icon: /images/icons/Technical_REALM_ConflictResolution3x.png
+      :icon-alt: An icon depicting conflict resolution
+
+      {+sync-short+} handles conflicts for you, so you don't have to write
+      complex custom code to resolve conflicting writes from multiple clients.
+      Read more: :ref:`conflict-resolution`
+
+
+.. kicker:: Where to Begin
+
+Get Started
+-----------
+
+.. card-group::
+   :columns: 2
+   :style: compact
+
+   .. card::
+      :headline: Launch a Template Starter App
+      :cta: Register for a free Atlas account
+      :url: https://www.mongodb.com/realm/register?tck=docs_CTA_app_services
+      :icon: /images/icons/Technical_REALM_iOSAndriodCrossPlatform3x.png
+      :icon-alt: An icon depicting two mobile devices synchronizing data between each other
+
+      The best way to start using {+sync-short+} is with a template starter app.
+      In the {+ui+}, under the :guilabel:`App Services` tab, you can select a
+      template. Atlas will configure a backend instance and give you the
+      frontend code for a selection of platforms. You can build your app on top
+      of this.
+
+   .. card::
+      :headline: Follow a {+sync-short+} Tutorial
+      :cta: Visit the Tutorials Page
+      :url: https://www.mongodb.com/docs/atlas/app-services/get-started/#tutorials
+      :icon: /images/icons/General_CONTENT_Tutorial3x.png
+      :icon-alt: A tutorial icon
+
+      The mobile app tutorials guide you through the process of building a
+      feature on top of a template starter app with {+sync+}.
+
+.. kicker:: Next Steps
+
+Explore the {+sync-short+} Documentation
+-------------------------------------
+
+.. procedure::
+
+   .. step:: Maintain Data Integrity with Sync Schemas
+
+      Define your application data model with standard schemas that you can use
+      to validate data and generate language-specific classes. Read the
```

Comment:
I wonder if we want to mention here that we can derive the schema from the client data model? Or if we think that's too much in-the-weeds for this page?

```suggestion
      Define your application data model with standard schemas that you can use
      to validate data. Generate language-specific classes, or infer the schema 
      from your client data model. Read the
```

Reply:
Yes, I think that's too far in the weeds.